<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingPage</class>
 <widget class="QWidget" name="SettingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>301</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkbox_daemon">
      <property name="text">
       <string>Run as Service</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkbox_refresh_passwd">
      <property name="text">
       <string>Auto refresh access token</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>391</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_relay">
        <property name="text">
         <string>Relay Server</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineedit_relay"/>
      </item>
      <item>
       <widget class="QPushButton" name="button_relay">
        <property name="text">
         <string>Confirm</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
